pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -671,6 +689,9 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 			volumeToMount.VolumeSpec.Name(),
 			volumeToMount.PodName,
 			volumeToMount.Pod.UID)
+		if errGetRef == nil && ref != nil {
+			oe.recorder.Eventf(ref, api.EventTypeNormal, kubecontainer.SuccessfulMountVolume, "MountVolume.SetUp succeeded for volume %q pod %q (UID: %q)", volumeToMount.VolumeName, volumeToMount.PodName, volumeToMount.Pod.UID)
+		}
 
 		// Update actual state of world
 		markVolMountedErr := actualStateOfWorld.MarkVolumeAsMounted(
```

Comment:
@saad-ali  -  yep, will add that in if everything looks good with what I have so far...